examples/token-auth-keycloak/conf/proxy/nginx.conf

```diff
@@ -21,58 +23,52 @@ server {
     proxy_buffering off;
     proxy_ignore_headers "X-Accel-Buffering";
 
+    #error_page  404              /404.html;
+
+    # redirect server error pages to the static page /50x.html
+    error_page   500 502 503 504  /50x.html;
+
+    root   /usr/share/nginx/html;
+    index  index.html index.htm;
+
     location /v2 {
-      # Do not allow connections from docker 1.5 and earlier
-      # docker pre-1.6.0 did not properly set the user agent on ping, catch "Go *" user agents
-      if ($http_user_agent ~ "^(docker\/1\.(3|4|5(?!\.[0-9]-dev))|Go ).*$" ) {
-          return 404;
-      }
-      proxy_set_header X-Forwarded-Proto $scheme;
-      proxy_set_header Host               $host;
-      proxy_set_header X-Forwarded-Host   $host;
-      proxy_pass $registry;
+        # Do not allow connections from docker 1.5 and earlier
+        # docker pre-1.6.0 did not properly set the user agent on ping, catch "Go *" user agents
+        if ($http_user_agent ~ "^(docker\/1\.(3|4|5(?!\.[0-9]-dev))|Go ).*$" ) {
+            return 404;
+        }
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host               $host;
+        proxy_set_header X-Forwarded-Host   $host;
+        proxy_pass $registry;
     }
 
     location /auth {
-      proxy_set_header X-Forwarded-Proto $scheme;
-      proxy_set_header Host               $host;
-      proxy_set_header X-Forwarded-Host   $host;
-      proxy_pass $keycloak;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host               $host;
+        proxy_set_header X-Forwarded-Host   $host;
+        proxy_pass $keycloak;
     }
 
     location /auth/realms/master/protocol/docker-v2/auth {
-      proxy_set_header X-Forwarded-Proto $scheme;
-      proxy_set_header Host               $host;
-      proxy_set_header X-Forwarded-Host   $host;
-      if ($http_authorization = "") {
-        add_header WWW-Authenticate 'Basic realm="Keycloak login"' always;
-        return 401;
-      }
-      proxy_pass $keycloak;
-    }
-
-    location /ui {
-      proxy_pass $ui;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host               $host;
+        proxy_set_header X-Forwarded-Host   $host;
+        if ($http_authorization = "") {
+            add_header WWW-Authenticate 'Basic realm="Keycloak login"' always;
+            return 401;
+        }
+        proxy_pass $keycloak;
```

Comment:
I use 2 spaces and not 4

Reply:
Tab 4 spaces standard for me in all projects) Ok i will use 2 spaces here :)